packages/grafana-data/src/transformations/standardTransformersRegistry.ts

```diff
@@ -28,8 +28,27 @@ export interface TransformerRegistryItem<TOptions> extends RegistryItem {
    * React component used as UI for the transformer
    */
   editor: React.ComponentType<TransformerUIProps<TOptions>>;
+
+  /**
+   * Set of categories associated with the transformer
+   */
+  categories?: Set<TransformerCategory>;
+
+  /**
+   * Image path for the transformer
+   */
+  image?: { dark: string; light: string };
 }
 
+export type TransformerCategory =
+  | 'combine'
+  | 'calculateNewFields'
+  | 'createNewVisualization'
```

Comment:
this is a very strange transformation category

Reply:
Thanks for the feedback! Do you mean the name of the group is odd, or the existence of the group itself is odd?

Some suggestions for names:
- Setup for visualization
- Adapt for visualization 
- Setup new visualization

As for the group itself, these are somewhat the odd ones out. We did a grouping exercise w/ some developers and this ultimately this is what I ended up with.